packages/react-magma-dom/src/components/CharacterCounter/CharacterCounter.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { InputMessage } from '../Input/InputMessage';
+import { magma } from '../../theme/magma';
+import { I18nContext } from '../../i18n';
+
+export interface CharacterCounterProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Total number of characters in an input.
+   */
+  inputLength?: number;
+  /**
+   * Sets the maximum amount of characters allowed.
+   */
+  isInverse?: boolean;
+  /**
+   * Enables the Character Counter and sets the maximum amount of characters allowed within the Input.
+   */
+  maxLength?: number;
```

Comment:
Should `inputLength` and `maxLength` be required props for this component? Without those props this component would be useless, right?

I think we want them to be required for CharacterCounter but optional for Input, Textarea, FormField container.